tests/preprocessing/test_warm_up_module.py

```diff
@@ -0,0 +1,161 @@
+import copy
+
+from fl4health.model_bases.apfl_base import ApflModule
+from fl4health.model_bases.fenda_base import FendaModel
+from fl4health.model_bases.moon_base import MoonModel
+from fl4health.preprocessing.warmed_up_module import WarmedUpModule
+from tests.test_utils.models_for_test import FeatureCnn, FendaHeadCnn, HeadCnn, SmallCnn, ToyConvNet, ToyConvNet_2
+
+
+def test_warm_up_module_loading_similar_models_without_mapping() -> None:
+    pretrained_model = ToyConvNet()
+    model = ToyConvNet()
+    old_model = copy.deepcopy(model)
+    warmupmodule = WarmedUpModule(pretrained_model=pretrained_model)
```

Comment:
Really minor: code style conventions typically would have this named `warmup_module` for readability throughout the test file.